.python-version

```diff
@@ -1 +0,0 @@
-3.9.7
```

Comment:
This file shouldn't be removed, it's for pyenv to use a specific python version.

Reply:
I would like to use a virtual environment with pyenv, with this file commited I can't. 

In general .python-version should be commited only for an application, not for a library: https://github.com/github/gitignore/blob/e5323759e387ba347a9d50f8b0ddd16502eb71d4/Python.gitignore#L85-L88. I don't think that devs using pyenv should be locked to a specific python version, if HWI itself supports many

Reply:
Anyway if it is a problem I can revert the change, it was only a nice touch (in my opinion) not essential for the PR